src/components/program-progress/ProgramProgressCourses.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useContext } from 'react';
+
+import PropTypes from 'prop-types';
+import {
+  Form, Col, Row,
+} from '@edx/paragon';
+import moment from 'moment';
+import { CheckCircle } from '@edx/paragon/icons';
+import { AppContext } from '@edx/frontend-platform/react';
+import { getEnrolledCourseRunDetails, getNotStartedCourseDetails } from './data/utils';
+
+const ProgramProgressCourses = ({ courseData }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  let coursesCompleted;
+  let coursesInProgress;
+  let coursesNotStarted;
+  const courseAboutPage = (key) => `/${enterpriseConfig.slug}/course/${key}`;
+
+  if (courseData?.completed) {
+    coursesCompleted = getEnrolledCourseRunDetails(courseData.completed);
+  }
+  if (courseData?.inProgress) {
+    coursesInProgress = getEnrolledCourseRunDetails(courseData.inProgress);
+  }
+  if (courseData?.notStarted) {
+    coursesNotStarted = getNotStartedCourseDetails(courseData.notStarted);
+  }
+  const { courseWithMultipleCourseRun, courseWithSingleCourseRun } = coursesNotStarted;
+
+  const renderCertificatePurchased = () => (
+    <Row className="d-flex align-items-start py-3 pt-5">
+      <Col className="d-flex align-items-center">
+        <span>Certificate Status: </span>
+        <CheckCircle className="fa fa-check-circle circle-color" />
+        <span>Certificate Purchased</span>
+      </Col>
+    </Row>
+  );
+  return (
+    <div className="col-8">
+      {coursesInProgress?.length > 0
+      && (
+        <div className="mb-5">
+          <h4>COURSES IN PROGRESS {coursesInProgress.length}</h4>
+          <hr />
+          {coursesInProgress.map((course) => (
+            (
+              <div className="mt-4.5 ml-4 pb-5">
+                <h4 className="text-dark-500">{course.title}</h4>
+                <p className="text-gray-500 text-capitalize mt-1">Enrolled:
+                  ({course?.pacingType.replace('_', '-')}) Starts {moment(course.start)
```

Comment:
nit: Isn't this supposed to be `Started` instead of `Starts`

Reply:
that was my assumption too. But that's how it is in designs.